src/views/mfa-verify/PassCodeForm.js

```diff
@@ -87,6 +102,13 @@ define(['okta', 'q', 'views/shared/TextBox'], function (Okta, Q, TextBox) {
         click: function () {
           form.clearErrors();
           this.disable();
+          if (factorType =='call' || factorType == 'sms') {
+            var warning = Okta.loc(factorType == 'call' ? 'factor.call.time.warning' : 'factor.sms.time.warning', 'login');
```

Comment:
we should add this to the response of the function `getFormAndButtonDetails`, that is where all these labels for sms or call are

Reply:
fixed